mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -118,6 +125,140 @@ case class ScalaPresentationCompiler(
 
   def didClose(uri: URI): Unit = {}
 
+
+  /// under development ↓
+
+  override def semanticTokens(
+      params: VirtualFileParams,
+      capableTypes: util.List[String],
+      capableModifiers: util.List[String]
+  ): CompletableFuture[ju.List[Integer]] = {
+
+    val empty: ju.List[Integer] = new ju.ArrayList[Integer]()
+    compilerAccess.withInterruptableCompiler(
+      empty,
+      params.token
+    ) { pc =>
+          new SemanticTokenProvider(
+              pc.compiler(),
+              params,capableTypes,capableModifiers
+            ).provide()
+    }
+  }
+
+  // override def semanticTokens(
```

Comment:
You can remove all the commented code now.

Reply:
I'll do it.